packages/react/src/components/TimePicker/TimePicker.types.ts

```diff
@@ -0,0 +1,44 @@
+import { IComboBoxProps } from '../../ComboBox';
+
+// Can only be numbers between 0-23
+export interface ITimeRange {
+  start: number;
+  end: number;
+}
+
+type PartialIComboBoxProps = Omit<IComboBoxProps, 'options'>;
+
+export interface ITimePickerProps extends PartialIComboBoxProps {
+  /**
+   * Label of the component
+   */
+  label?: string;
+  /**
+   * Time increments, in minutes, of the options in the dropdown
+   */
+  increments?: number;
+  /**
+   * Whether to show seconds in the component
+   */
+  showSeconds?: boolean;
+  /**
+   * Whether to use 12 or 24 hour time //
+   */
+  useHour12?: boolean;
+  /**
+   * Toggle to allow user input
```

Comment:
Revised comment to now be
`If true, the TimePicker allows freeform user input, rather than restricting to the default increments. The input will still be restricted to valid time values.`